src/app/Shared/Context/DataContext.tsx

```diff
@@ -7,13 +7,15 @@ interface DataInterface {
     ZNDataFiltered: DbCollection;
     keyValues: DbKey[];
     uniqueValues: FilterInterface;
+    nrOfSelectedIsolates: number;
```

Comment:
Is this always the same as ZNDataFiltered.length?

Reply:
Yes it is.

Reply:
Now that ZNDataFiltered is removed: Is this always the same as ZNData.length? If so, i don't understand what's the use of the nrOfSelectedIsolates

Reply:
Question answered in stand-up.